charts/osm/values.schema.json

```diff
@@ -0,0 +1,283 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "title": "The OSM Helm chart values schema",
+    "required": [
+        "OpenServiceMesh"
+    ],
+    "properties": {
+        "OpenServiceMesh": {
+            "$id": "#/properties/OpenServiceMesh",
+            "type": "object",
+            "title": "The OpenServiceMesh schema",
+            "description": "The top level required key in the values file.",
+            "default": {},
```

Comment:
I noticed the example in Helm's docs doesn't include defaults. How much value is there in adding them here? And what could go wrong if these defaults get out of sync with values.yaml?

Reply:
That's a really good point. I had them in here because they're an option in json schema but Helm is not using them. Let me go ahead and take them out.